backend/module/chaos/serverexperimentation/rtds/rtds.go

```diff
@@ -11,10 +11,15 @@ import (
 	"time"
 
 	gcpV2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
-	gcpCore "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
-	gcpDiscovery "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
-	gcpCache "github.com/envoyproxy/go-control-plane/pkg/cache/v2"
-	gcpServer "github.com/envoyproxy/go-control-plane/pkg/server/v2"
+	gcpCoreV2 "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
+	gcpCoreV3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	gcpDiscoveryV2 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+	gcpDiscoveryV3 "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v3"
+	gcpDiscoveryServiceV3 "github.com/envoyproxy/go-control-plane/envoy/service/runtime/v3"
```

Comment:
nit : can we rename this to reflect `runtime` in it? Maybe gcpRuntimeDiscoveryServer as gcpRTDS